### PR TITLE
small edits while reading the code

### PR DESCRIPTION
**Pull request by akiezun**
@lbergelson please review. minor edits

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipeline.java [hunk 1]
```diff
@@ -103,26 +105,29 @@ final protected void setupPipeline(Pipeline pipeline) {
     }
 
     @VisibleForTesting
-    protected PCollection<?> applyTransformsToPipeline(String headerString, PCollection<Read> preads) {
-        for (ReadFilter filter : getReadFilters()) {
+    protected PCollection<?> applyTransformsToPipeline(final String headerString, final PCollection<Read> preadsArgs) {
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipeline.java]: I don't understand this rename.  This thing coming in is actually the PCollection of Reads, not arguments.

File: src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipelineTest.java [hunk 2]
```diff
@@ -71,9 +73,9 @@ public RuntimeConfigurablePipeline(PTransformSAM<?> tool, List<ReadFilter> filte
                 new Object[] { Lists.newArrayList(none), Collections.emptyList(), 0},
                 new Object[] { Collections.<ReadFilter>emptyList(), Lists.newArrayList(replaceBasesWithA), 7},
                 new Object[] { Lists.newArrayList(all), Lists.newArrayList(replaceBasesWithA), 7},
-                new Object[] { Lists.newArrayList(all), Lists.newArrayList(replaceBasesWithA.andThen(ReadTransformer.identity())), 7},
-                new Object[] { Lists.newArrayList(none.negate().and(all)), Lists.newArrayList(replaceBasesWithA.compose(ReadTransformer.identity())), 7},
-                new Object[] { Lists.newArrayList(none.negate(),all, all), Lists.newArrayList(replaceBasesWithA, replaceBasesWithA, ReadTransformer.identity()), 7}
+                new Object[] { Lists.newArrayList(all), Lists.newArrayList(replaceBasesWithA.andThen(UnaryOperator.identity())), 7},
```
Comment (lbergelson) [inline src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipelineTest.java]: Is this change necessary/correct?  Intellij erroneously reports a compile error sometimes here, but I think we actually do want ReadTransformer.identity here.

**Comment by lbergelson** [comment]
@akiezun Failing due to that ReadTransformer change.  Pretty sure those Red underlines you saw are an intellij bug with java 8.  We should probably report it.

**Comment by lbergelson** [comment]
@akiezun Back to you with what I think is a legitimate test failure.

File: src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipeline.java (hunk 1 above)
Comment (akiezun) [inline src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipeline.java]: the previous code assigned to a method parameter which is bad style. I renamed it now to 'preadsIn' to not be confused with 'arguments'

Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipeline.java]: Good point on the assignment.

**Comment by akiezun** [comment]
back to @lbergelson

**Comment by lbergelson** [comment]
:+1: It has some conflict with master now though.  Merge when ready.

**Comment by lbergelson** [comment]
@akiezun Back to you.
